cluster/sync.sh

```diff
@@ -41,16 +32,25 @@ function consistently {
     done
 }
 
-function isHandlerOk {
-    desiredNumberScheduled=$(getDesiredScheduledHandlers $1)
-    numberAvailable=$(getAvailableHandlers $1)
+function isDaemonSetOk {
+
+    desiredNumberScheduled=$($kubectl get daemonset -n $1 -l $2 -o=jsonpath='{..status.desiredNumberScheduled}')
```

Comment:
Why do we have two dots in the jsonpath?

Reply:
To simplify expression and just read the first occurrence of that field now we have just one DaemonSet